src/pages/our-work.astro

```diff
@@ -160,16 +154,20 @@ import Timeline from "../components/Timeline.astro";
 							description:
 								"Partnering with incredible local candidates like Ambria Taylor and Anthony Quezada, we launched a series of highly optimized websites for candidates running in hyper-local races around the country.",
 						},
-						{
-							title: "Supporting Zephyr Teachout's run for Attorney General.",
-							description:
-								"Ut enim aeque doleamus animo, cum corpore dolemus, fieri tamen permagna accessio potest, si aliquod aeternum et infinitum impendere malum nobis opinemur.",
-						},
 					],
 				},
 				{
 					year: "2021",
 					items: [
+						{
+							title: "Supporting Zephyr Teachout's run for Attorney General.",
+							description:
+								"Zephyr Teachout is a respected leader in exposing corporate monopolies, fighting political corruption, and championing policies that empower working people. We were proud to support her campaign for NY Attorney General.",
```

Comment:
I wonder if we should mention it was short-lived?

Reply:
mm i dunno if it's necessary tbh. do you have a particular rationale for that?

Reply:
Nah, probably fine! Let's leave it.